Update with Russ's comments

mingpeiwk opened the pull request:
Updated with Russ's comments and agreed revision on "Trust Anchor" in a device and definition of "App Store".

mingpeiwk committed:
Update with Russ's comments

mingpeiwk committed:
Typo fix

dthaler commented on draft-ietf-teep-architecture.md:
```diff
@@ -228,6 +228,8 @@ location where another domain-specific protocol standard (e.g., {{GSMA}},
 
 The following terms are used:
 
+  - App Store: an online location from which Untrusted Applications can be downloaded.
```
```suggestion
  - App Store: An online location from which Untrusted Applications can be downloaded.
```

dthaler reviewed (approved):
Ok except for capitalization nit

mingpeiwk committed:
Update draft-ietf-teep-architecture.md

Co-authored-by: Dave Thaler <dthaler@microsoft.com>

mingpeiwk committed:
Restore the non-root Trust Anchor citation

Restore back to the original Trust Anchor that allows to be a non-root certificate for TAM too.

mingpeiwk committed:
Updated Trust Anchor definition

Extend the format reference to include "other structure".

dthaler commented on draft-ietf-teep-architecture.md:
```diff
@@ -638,9 +641,8 @@ the Untrusted Application and its corresponding TAs are packaged, delivered, and
-Implementations must support encryption of such
-Personalization Data to preserve the confidentiality and integrity protection of potentially
-sensitive data contained within it.
+Implementations must support encryption to preserve the confidentiality and integrity protection of such Personalized Data,
```
Delete "protection"

dthaler commented on draft-ietf-teep-architecture.md:
```diff
@@ -1287,7 +1289,8 @@ for protecting the resource usage allocated for Trusted Component management.
 
 A root CA for TAM certificates might get compromised or its certificate might
-expire.
+expire, or a Trust Anchor other than a root CA certificate may also expire or
+be compromised.
```
I see this was added back in after 234, and this is fine.
